src/main/java/com/github/dockerjava/core/command/CreateContainerCmdImpl.java

```diff
@@ -611,6 +611,13 @@ public CreateContainerCmd withCpuShares(Integer cpuShares) {
     }
 
     @Override
+    public CreateContainerCmd withPidsLimit(Long pidsLimit) {
+        checkNotNull(pidsLimit, "pidsLimit was not specified");
+        hostConfig.withPidsLimit(pidsLimit);
+        return this;
```

Comment:
So even in testcase you create and configure hostConfig directly without calling proxy methods.